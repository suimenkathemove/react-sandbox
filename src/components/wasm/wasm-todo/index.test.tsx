import { cleanup, render } from '@testing-library/react';

import { defaultProps } from './index.stories';

import { WasmTodo } from './';

const setup = () => render(<WasmTodo {...defaultProps} />);

describe('WasmTodo', () => {
  afterEach(() => {
    cleanup();
  });

  test('render', () => {
    const { asFragment } = setup();
    expect(asFragment()).toMatchSnapshot();
  });
});
