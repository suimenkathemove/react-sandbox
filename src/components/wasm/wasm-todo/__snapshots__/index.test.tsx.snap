// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WasmTodo render 1`] = `
<DocumentFragment>
  <div>
    <button>
      show
    </button>
    <button>
      create
    </button>
    <button>
      update
    </button>
    <button>
      delete
    </button>
  </div>
</DocumentFragment>
`;
